src/hooks/help/useHelp.ts

```diff
@@ -54,8 +55,6 @@ const useHelpState = () =>
                 newReport.messageId = options.messageId;
                 newReport.currentStep = ReportState.SELECT_TOPICS;
                 break;
-            case ReportType.PHOTO:
```

Comment:
For a photo report we should need: activeReport.extraData (which is the photo url for photo furnitures), activeReport.cfhTopic, roomId, and the reported photo furniture id